doc/getting_started.md

```diff
@@ -7,6 +7,26 @@ In the meantime, check the following PRs:
 * https://github.com/lowRISC/opentitan/pull/2587
 * https://github.com/lowRISC/opentitan/pull/2735
 
+### Downloading code via FTDI
+
+Currently the target firmware is downloaded using the
+[spiflash](https://docs.opentitan.org/sw/host/spiflash/README/) utility. A
+debian-compatible pre-built binary is available at cw/cw305/bin/linux/spiflash.
```

Comment:
I understand that it is convenient to "ship" this and other binaries (bitstreams) directly with the code in the same repo. I also think this is okay now that we are at the very beginning as it eases collaboration and allows to get up to speed more easily.

However, I don't think we should keep updating these binaries. Some of them are pretty big and they will remain in the tree forever. Once we have things up and running, I suggest we kick the binaries out again and create a script to download them from GitHub releases. There we can periodically provide new binaries as we do with the toolchains https://github.com/lowRISC/lowrisc-toolchains/releases.

Reply:
SGTM. That's a good idea :)